projects/hslayers/src/components/styles/styles.module.ts

```diff
@@ -1,10 +1,13 @@
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import {CUSTOM_ELEMENTS_SCHEMA, NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
+import {DragDropModule} from '@angular/cdk/drag-drop';
 import {FormsModule} from '@angular/forms';
-import {TranslateModule} from '@ngx-translate/core';
+import {MatExpansionModule} from '@angular/material/expansion';
```

Comment:
I dont like to mix material design into our bootstrap version of hslayers. Becomes too much of a Frankenstein.

Reply:
I understand, but couldn't get drag and drop to work on ngb

Reply:
I will try to make it work with ngb then.

Reply:
How ever we could consider to move to angular-material at some point and remove ngb dependencies. (Just a thought)

Reply:
I think the correct way would be to have a UI kit agnostic hslayers core and then have 2 more extensions/libraries - one for NgBootstrap one for Material